http/oidc/src/main/java/org/wildfly/security/http/oidc/OidcClientContext.java

```diff
@@ -525,6 +525,107 @@ public String getTokenSignatureAlgorithm() {
         public void setTokenSignatureAlgorithm(String tokenSignatureAlgorithm) {
             delegate.setTokenSignatureAlgorithm(tokenSignatureAlgorithm);
         }
+
+        @Override
+        public String getAuthenticationRequestFormat() {
+            return delegate.getAuthenticationRequestFormat();
+        }
+
+        @Override
+        public void setAuthenticationRequestFormat(String authFormat) {
+            delegate.setAuthenticationRequestFormat(authFormat);
+        }
+
+        @Override
+        public String getRequestSignatureAlgorithm() {
+            return delegate.getRequestSignatureAlgorithm();
+        }
+
+        @Override
+        public void setRequestSignatureAlgorithm(String requestSignature) {
+            delegate.setRequestSignatureAlgorithm(requestSignature);
+        }
+
+        @Override
+        public String getRequestEncryptAlgorithm() {
+            return delegate.getRequestEncryptAlgorithm();
+        }
+
+        @Override
+        public void setRequestEncryptAlgorithm(String algorithm) {
+            delegate.setRequestEncryptAlgorithm(algorithm);
+        }
+
+        @Override
+        public String getRequestContentEncryptionMethod() {
+            return delegate.requestContentEncryptionMethod;
+        }
+
+        @Override
+        public void setRequestContentEncryptionMethod (String enc) {
+            delegate.requestContentEncryptionMethod = enc;
+        }
+
+        @Override
+        public String getRequestObjectSigningKeystoreFile() {
+            return delegate.requestObjectSigningKeystoreFile;
+        }
+
+        @Override
+        public void setRequestObjectSigningKeystoreFile(String keyStoreFile) {
+            delegate.requestObjectSigningKeystoreFile = keyStoreFile;
+        }
+
+        @Override
+        public String getRequestObjectSigningKeyStorePassword() {
```

Comment:
We should be consistent with Keystore vs KeyStore in the method names in this file.